<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="detect_track2D_and_locate">
    <rosparam file="$(find detect_and_track)/config/object_detection_rock.yaml"/>
    <rosparam file="$(find detect_and_track)/config/image_tracker_rock.yaml"/>
  </group>

  <node pkg="tf" type="static_transform_publisher" name="optitrack_to_robot" args="0.0 0.0 0.0 0.0 0.0 0.0 1.0 robot base_link 10" />

  <node name="track2D" pkg="detect_and_track" type="track2D_node" output="screen">
    <remap from="/camera/color/image_raw" to="/camera01/color/image_raw"/>
  </node>
</launch>